orbit/pkg/update/notifications.go

```diff
@@ -501,14 +501,14 @@ func (w *windowsMDMBitlockerConfigFetcher) attemptBitlockerEncryption(notifs fle
 		ClientError:   bitlockerError,
 	}
 
+	err = w.EncryptionResult.SetOrUpdateDiskEncryptionKey(payload)
 	if err != nil {
-		log.Error().Err(err).Msg("failed to encrypt the volume")
+		log.Error().Err(err).Msg("failed to send encryption result to Fleet Server")
 		return
 	}
 
-	err = w.EncryptionResult.SetOrUpdateDiskEncryptionKey(payload)
 	if err != nil {
-		log.Error().Err(err).Msg("failed to send encryption result to Fleet Server")
+		log.Error().Err(err).Msg("failed to encrypt the volume")
```

Comment:
`err` here is set at `recoveryKey, err := fn(targetVolume)`

Reply:
Just to clarify, I meant a comment in the code? :D